storage/minio/src/test/unit/backend/RunCommonUnitTests.ts

```diff
@@ -0,0 +1,69 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import "reflect-metadata";
+
+import { Container } from "inversify";
+import { Client } from "minio";
+import { createStubInstance } from "sinon";
+
+import {
+  rebindS3Client,
+  rebindS3Server,
+  s3TestConfig,
+} from "@itwin/object-storage-s3/lib/test/backend/CommonUnitTestUtils";
+
+import { S3ServerStorageBindingsConfig } from "@itwin/object-storage-s3";
+import { StorageUnitTests } from "@itwin/object-storage-tests-backend-unit";
+
+import { MinioClientStorageBindings } from "../../../client";
+import { MinioServerStorageBindings } from "../../../server";
+
+const dependencyName = "minio";
+const minioTestConfig = {
+  // eslint-disable-next-line @typescript-eslint/naming-convention
+  ServerStorage: {
+    ...s3TestConfig.ServerStorage,
+    dependencyName,
+  },
+  // eslint-disable-next-line @typescript-eslint/naming-convention
+  ClientStorage: {
+    dependencyName,
+  },
+  // eslint-disable-next-line @typescript-eslint/naming-convention
+  FrontendStorage: {
+    dependencyName,
+  },
+};
+
+class TestMinioServerStorageBindings extends MinioServerStorageBindings {
+  public override register(
+    container: Container,
+    config: S3ServerStorageBindingsConfig
+  ): void {
+    super.register(container, config);
+    rebindS3Server(container);
+
+    const mockClient = createStubInstance(Client);
+
+    container.rebind(Client).toConstantValue(mockClient);
+  }
+}
+
+class TestMinioClientStorageBindings extends MinioClientStorageBindings {
+  public override register(container: Container): void {
+    super.register(container);
+    rebindS3Client(container);
+  }
+}
+
+const tests = new StorageUnitTests(
+  minioTestConfig,
+  TestMinioServerStorageBindings,
+  TestMinioClientStorageBindings
+);
+tests.start().catch((err) => {
+  process.exitCode = 1;
+  throw err;
+});
```

Comment:
I think if there was no empty line github would show this - 
![image](https://user-images.githubusercontent.com/70565417/167999415-73b0d5bd-051b-4b43-a70c-d3a1babc78bb.png)
Since there is no such symbol the empty line is there, it is also visible in locally checked-out repo. I would personally prefer if there was no trickery in regards to displaying empty lines at the end of file..

Empty lines at the end of `.ts` files are enforced by `prettier` ESLint plugin, linting does result in an error if no newline is present.